shared/common-adapters/new-input.tsx

```diff
@@ -66,7 +60,7 @@ class ReflessNewInput extends React.Component<Props & RefProps, State> {
         {!!this.props.icon && (
           <Box style={styles.icon}>
             <Icon
-              color={globalColors.black_20}
+              color={Styles.globalColors.black_20} // not sure how to make this dynamic
```

Comment:
@chrisnojima did you have a plan for icon inverting?

Reply:
this will automatically get inverted since its in a render